scripts/seed_data.py

```
@@ -0,0 +1,109 @@
+import json
+import os
+from django.utils import timezone
+from django.db import transaction
+
+from accounts.models import User, Profile
+from classrooms.models import Classroom, ClassroomTeacher, Enrollment
+
+
+@transaction.atomic
+def create_entities(data):
```

Ruff (0.8.2)

Missing return type annotation for public function `create_entities`

Add return type annotation: `None`

(ANN201)